components/Alerts/Alerts.css

```diff
@@ -0,0 +1,57 @@
+
+.alerts{
+    width:90%;
+    padding:1rem;
+    font-weight: bold;
+    border-radius:3px;
+    margin:0.5rem 0;
+    
+}
+.primary__alert{
+    background-color: rgb(127,210,230);
+    color:rgb(41,78,168);
+}
+.sucess__alert{
+    background-color: rgb(132,241,201);
+    color:rgb(27,112,82);
+}
+.danger__alert{
+    background-color: rgb(252,197,197);
+    color:rgb(220,53,69);
+    
```

Comment:
Create variables for color should be helpful.

Reply:
Okay..will work on it...thanks.